tests/foreman/ui_airgun/test_container.py

```diff
@@ -4,13 +4,11 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
-
```

Comment:
`CaseLevel: Acceptance ` removing from module level? We don't want to keep some default?

Reply:
That value is overwritten for every single test in this file.

Then, these tests are covering feature that is removed in 6.6, so they are going to be removed anyway. Probably whole file will be deleted.

And finally, if anyone ever adds new test there and forgets to set CaseLevel, `testimony` will complain about missing required token. So they will know to set CaseLevel for their test, or set default for entire file.

If these arguments don't convince you, let me know, I will add this line back.

Reply:
Hmm, make sense! leave it then